go.mod

```diff
@@ -4,8 +4,8 @@ go 1.21.5
 
 require (
 	github.com/avast/retry-go/v4 v4.5.1
-	github.com/bitcoin-sv/spv-wallet-go-client v1.0.0-beta.2
-	github.com/bitcoin-sv/spv-wallet/models v1.0.0-beta.2
+	github.com/bitcoin-sv/spv-wallet-go-client v1.0.0-beta.2.0.20240517103917-ab378a15bfea
```

Comment:
Remember to release new `go-client` and bump its version here. 
After PR approvals and testing, of course.

Reply:
well yes someone would need to show me or would need to trigger this release

Reply:
sure 😄 👍